cmd/pctl/install.go

```diff
@@ -59,6 +58,12 @@ func installCmd() *cli.Command {
 				DefaultText: "main",
 				Usage:       "The base branch to open a PR against.",
 			},
+			&cli.StringFlag{
+				Name:        "out",
+				DefaultText: "current",
+				Value:       ".",
+				Usage:       "Optional location to output generated files into.",
+			},
```

Comment:
its slightly misleading, its "the location to create the profile installation directory in", as opposed to the place where the contents of the profile installation go in

Reply:
ah, true, that was before I realised it creates a folder. :D